electrum/base_wizard.py

```diff
@@ -489,6 +538,34 @@ def f(derivation, script_type):
                 if hasattr(client, 'clear_session'):  # FIXME not all hw wallet plugins have this
                     client.clear_session()
                 raise
+        elif purpose == HWD_SETUP_NEW_BTCV_WALLET:
+            if 'recovery_password' not in self.data:
+                raise Exception('Recovery password not set')
+
+            self.plugin.set_recovery_password(device_info.device.id_, self.data['recovery_password'], self)
+            if 'instant_password' in self.data:
+                self.plugin.set_instant_password(device_info.device.id_, self.data['instant_password'], self)
+            else:
+                self.plugin.set_instant_password(device_info.device.id_, str(0x00), self)
+
+            derivation = bip44_derivation(0)
+            script_type = 'p2wsh-p2sh'
+            self.run('on_hw_derivation', name, device_info, derivation, script_type)
+        elif purpose == HWD_SETUP_DECRYPT_BTCV_WALLET:
+            if 'recovery_password' not in self.data:
+                raise Exception('Invalid recovery password')
```

Comment:
Shouldn't it be similar to the line 542?

Reply:
It's 3 keys wallet import, line 542 in wallet creation